### PR TITLE
Update for 1.2.4

### PR DESCRIPTION
**Pull request by MasterHW**
1.2.4 includes ability to natively use bcnode's external miner (https://github.com/trick77/bcnode-gpu-docker/pull/10). Update the images to properly use this.

### CODE REVIEW

File: k8s/bcnode/deployment.yml [hunk 1]
```diff
@@ -86,16 +88,16 @@ spec:
           name: vol-bcnode-db
         command: ['sh','-c']
         args:
-        - if [ ! -d "/data/db" ]; then
+        - rm -r /data/db;
```
Comment (lgray) [inline k8s/bcnode/deployment.yml:91]: Revert the changes lines 91 - 103. `--strip-components=2` is for the tarball output of export-db.sh from bcnode-gpu-docker, while `--strip-components=1` is for the "easysync" zipballs. A better solution would be to (in a different PR) update `upload-db.sh` to forward the file extension and deal with it appropriately here.

**Comment by lgray** [comment]
Thanks for pushing this! I just haven't had the time to get around to the chore.

I have one review comment, please address it.

File: k8s/bcnode/deployment.yml (hunk 1 above)
Comment (lgray) [reply k8s/bcnode/deployment.yml:91]: Also the default behavior of deleting the chain state on restart is an enormous time waster. Prefer that we instead tell people how to refresh the chainstate. There are a few ways.

Comment (MasterHW) [reply k8s/bcnode/deployment.yml]: Thanks, reverted.
